CODEOWNERS

```diff
@@ -0,0 +1,103 @@
+/dpsim-models/include/dpsim-models/DP/DP_Ph1_SynchronGenerator6bOrderVBR.h @martinmoraga
+/dpsim-models/include/dpsim-models/DP/DP_Ph1_SynchronGeneratorTrStab.h @gnakti
+/dpsim-models/include/dpsim-models/DP/DP_Ph1_varResSwitch.h @gnakti
+/dpsim-models/include/dpsim-models/EMT/EMT_Ph3_SynchronGenerator3OrderVBR.h @martinmoraga
+/dpsim-models/include/dpsim-models/EMT/EMT_Ph3_SynchronGenerator4OrderVBR.h @martinmoraga
+/dpsim-models/include/dpsim-models/EMT/EMT_Ph3_SynchronGenerator5OrderVBR.h @martinmoraga
+/dpsim-models/include/dpsim-models/EMT/EMT_Ph3_SynchronGenerator6aOrderVBR.h @martinmoraga
+/dpsim-models/include/dpsim-models/EMT/EMT_Ph3_SynchronGenerator6bOrderVBR.h @martinmoraga
+/dpsim-models/include/dpsim-models/SP/SP_Ph1_SynchronGenerator3OrderVBR.h @martinmoraga
+/dpsim-models/include/dpsim-models/SP/SP_Ph1_SynchronGenerator4OrderVBR.h @martinmoraga
+/dpsim-models/include/dpsim-models/SP/SP_Ph1_SynchronGenerator5OrderVBR.h @martinmoraga
+/dpsim-models/include/dpsim-models/SP/SP_Ph1_SynchronGenerator6aOrderVBR.h @martinmoraga
+/dpsim-models/include/dpsim-models/SP/SP_Ph1_SynchronGenerator6bOrderVBR.h @martinmoraga
+/dpsim-models/include/dpsim-models/SP/SP_Ph1_SynchronGeneratorTrStab.h @gnakti
+/dpsim-models/include/dpsim-models/SP/SP_Ph1_varResSwitch.h @gnakti
+/dpsim-models/src/Base/Base_ReducedOrderSynchronGenerator.cpp @martinmoraga
+/dpsim-models/src/CIM/Reader.cpp @gnakti
+/dpsim-models/src/DP/DP_Ph1_SynchronGenerator3OrderVBR.cpp @martinmoraga
+/dpsim-models/src/DP/DP_Ph1_SynchronGenerator4OrderVBR.cpp @martinmoraga
+/dpsim-models/src/DP/DP_Ph1_SynchronGenerator5OrderVBR.cpp @martinmoraga
+/dpsim-models/src/DP/DP_Ph1_SynchronGenerator6aOrderVBR.cpp @martinmoraga
+/dpsim-models/src/DP/DP_Ph1_SynchronGenerator6bOrderVBR.cpp @martinmoraga
+/dpsim-models/src/DP/DP_Ph1_SynchronGeneratorTrStab.cpp @gnakti
+/dpsim-models/src/DP/DP_Ph1_varResSwitch.cpp @gnakti
+/dpsim-models/src/EMT/EMT_Ph3_SynchronGenerator3OrderVBR.cpp @martinmoraga
+/dpsim-models/src/EMT/EMT_Ph3_SynchronGenerator4OrderVBR.cpp @martinmoraga
+/dpsim-models/src/EMT/EMT_Ph3_SynchronGenerator5OrderVBR.cpp @martinmoraga
+/dpsim-models/src/EMT/EMT_Ph3_SynchronGenerator6aOrderVBR.cpp @martinmoraga
+/dpsim-models/src/EMT/EMT_Ph3_SynchronGenerator6bOrderVBR.cpp @martinmoraga
+/dpsim-models/src/SP/SP_Ph1_SynchronGenerator3OrderVBR.cpp @martinmoraga
+/dpsim-models/src/SP/SP_Ph1_SynchronGenerator4OrderVBR.cpp @martinmoraga
+/dpsim-models/src/SP/SP_Ph1_SynchronGenerator5OrderVBR.cpp @martinmoraga
+/dpsim-models/src/SP/SP_Ph1_SynchronGenerator6aOrderVBR.cpp @martinmoraga
+/dpsim-models/src/SP/SP_Ph1_SynchronGenerator6bOrderVBR.cpp @martinmoraga
+/dpsim-models/src/SP/SP_Ph1_SynchronGeneratorTrStab.cpp @gnakti
+/dpsim-models/src/SP/SP_Ph1_varResSwitch.cpp @gnakti
+/dpsim-villas/ @stv0g
+/dpsim/examples/cxx/CIM @gnakti
+/dpsim/examples/cxx/Circuits @gnakti
+/dpsim/examples/cxx/Circuits/DP_ReducedOrderSG_SMIB_Fault.cpp
```

Comment:
The owner is missing here @stv0g

Reply:
Oups, fixed. I was blindly copying @martinmoraga proposal.